pkg/v1/providers/vendir.yml

```diff
@@ -23,6 +23,6 @@ directories:
   - path: .
     git:
       url: git@github.com:vmware-tanzu/community-edition.git
-      ref: 1ce18be43347405d3dfbde4ba2cbb0d1e2fa0b41
+      ref: 20dc640340aaef71c9e643f5faab65df50266dbf
```

Comment:
hi @lubronzhan , I had a ques regarding this change. Why do we need to update the SHA here for kapp-controller
Changes for new features (schema) are in TCE and downstream repo for packages
cc: @blc1996

Reply:
Oh is that's the reason I'm pulling the newest change of OPENV3API changes?

Reply:
The change in this line is not needed. It will update the manifest for kapp-controller. You just need to update the line for CPI.

Reply:
Yes thats why you were pulling the latest change. You can undo that line change